modules/network-observability-without-loki.adoc

```diff
@@ -0,0 +1,19 @@
+// module included in the following assemblies:
+// networking/network_observability/installing-operators.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-without-loki{context}"]
+= Network Observability without Loki
+You can use Network Observability without Loki by omitting the Loki installation steps and only installing Kafka. The following table compares available features with and without Loki:
+
+.Comparison of feature availability with and without Loki
+[options="header"]
+|============================================================================================================================
+|                                     | *With Loki* | *Without Loki* 
+| *Exporters*                         | X           | X      
+| *Flow-based dashboards*             | X           | X       
+| *Traffic Flow and Topology views*   | X           | N/A     
+| *Quick Filters*                     | X           | N/A    
+| *Web console integration*           | X           | N/A
```

Comment:
```suggestion
| *OCP console Network Traffic tab integration*           | X           | N/A
```

What about a more explicit description here ?

Reply:
Good w/ me

Reply:
@jpinsonneau I guess this row is basically the same as the *Traffic Flow Overview, Table and Topology views* row. Do you have a preference for how we describe this? Or do you think both are necessary?

Reply:
These 2 are parts of the console plugin but are actually different:
- Overview / Table / Topology are `views` we provide with the plugin
- Network Traffic tab integration are `tab entry` added to kubernetes objects (Pod / Service / DeamonSet ...) pages and bringing the user to our `views` with proper filtering